metron-platform/metron-enrichment/src/main/java/org/apache/metron/enrichment/adapters/jdbc/JdbcAdapter.java

```diff
@@ -38,6 +39,27 @@
   private JdbcConfig config;
   private String host;
 
+  protected boolean isConnectionClosed() {
+    boolean isClosed = statement == null || connection == null;
+    if(!isClosed) {
+      try {
+        isClosed = statement.isClosed() || connection.isClosed();
+      } catch (SQLException e) {
+        _LOG.error("Unable to maintain open JDBC connection: " + e.getMessage(), e);
+        isClosed = true;
+      }
+    }
+    return isClosed;
+  }
+
+  protected boolean resetConnectionIfNecessary() {
+    if(isConnectionClosed())
+    {
```

Comment:
So, I'll definitely make the class consistently K&R (I must've fat-fingered the newline).  We don't have a coherent style at the moment in this respect.  I prefer K&R bracing, so you'll see that in my code, but yeah, I agree, we need to correct style as we see it.

Also, don't worry about nitpicking..we all have to live in this code. :)